VstsTasks/docs/overview.md

```diff
@@ -1,74 +1,119 @@
+
+**Number of times to retry the deployment** (optional) - Number of times to retry the deployment when an error occurs, either while creating the lab VM or if any artifact fails to apply successfully.
+
+### Output Variables
+The task can produce the following outputs into corresponding variables:
+
+**Lab VM ID** - Variable to capture the created lab VM ID. Default is `labVMId`. The variable can be referred as `$(labVMId)` in subsequent tasks. You can use a name other than the default.
+
+## Delete VM
+![Delete VM](screenshots/azure-dtl-deletevm.png)
+The task allows you to delete a lab VM.
+### Input Parameters
+The task requires the following inputs: 
+
+**Azure RM Subscription** - Azure Resource Manager subscription to configure before running. 
+
+**Lab VM ID** - Resource ID of the lab VM to delete. Default is `$(labVMId)`. You can use any variable such as `$(labVMId)`, which is the output of calling **Create Azure DevTest Labs VM**, that contains a value in the form `/subscriptions/{subId}/resourceGroups/{rgName}/providers/Microsoft.DevTestLab/labs/{labName}/virtualMachines/{vmName}`.
+
+## Create Environment
+![Create Environment](screenshots/azure-dtl-createenvironment.png)
+The task allows you to create an Environment.
+### Input Parameters
+The task requires the following inputs:
+
+**Azure RM Subscription** - Azure Resource Manager subscription to configure before running.
+
+**Lab Name** - Azure DevTest Lab name.
+
+**Repository Name** - Repository friendly name where the environment ARM templates are stored.
+
+**Template Name** - Name of the environment ARM template.
+
+**Environment Name** - Name of the deployed template.
+
+**Parameters File (optional)** - The input parameters file. Use either a parameters file or the parameters.
+
+**Parameters** - The individual parameters.
+
+**Create output variables based on the environment template output.** - Create output variables resulting from the creation of the environment. See Output Variables for details. 
+
+### Output Variables
+If enabled, the task can produce the following output variables.  Note that you will need to define a Reference Name under the Output Variables section to correctly reference the variables in the list.
+
+**EnvironmentResourceId** - Resource ID of the environment created. Usage $(<refName>.environmentResourceId).
+
+**EnvironmentResourceGroupId** - Resource ID of the resource group for the environment created. Usage $(<refName>.environmentResourceGroupId).
+
+## Delete Environment
+![Delete Environment](screenshots/azure-dtl-deleteenvironment.png)
+The task allows you to delete an Environment.
+### Input Parameters
+The task requires the following inputs.
+
+**Azure RM Subscription** - Azure Resource Manager subscription to configure before running.
+
+**Lab Name** - Azure DevTest Lab name.
+
+**Environment Name** - Name of the environment to be deleted.

```

Comment:
Change to the following:

>**Environment Name or ID** - Either the name of the environment to be deleted, selected from the dropdown list; or, the fully qualified resource ID of the environment to be deleted. This can be a reference to the output variable from a call to the Create Environment task. For example, `$(<refName>.environmentResourceId)`.